runtime/libs/nnapi/include/NeuralNetworksShim.h

```diff
@@ -1130,7 +1130,7 @@ inline int ANeuralNetworksExecution_getDuration(const ANeuralNetworksExecution *
   LOAD_FUNCTION(ANeuralNetworksExecution_getDuration);
   EXECUTE_FUNCTION_RETURN(execution, durationCode, duration);
 }
-
+#if __ANDROID_API__ >= 30
```

Comment:
Some APIs below this macro says `Available since API level 29`. I think this macro should be moved to line 1221.

Reply:
Oh Thanks. I will check it.